<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timesheet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="EntryListView" name="listView" native="true">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="alternatingRowColors" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="dateLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="fromLineEdit">
             <property name="placeholderText">
              <string>od</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Time:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&lt;b&gt;Date:&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="toLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="placeholderText">
              <string>do</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="titleTextEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>64</height>
            </size>
           </property>
           <property name="tabChangesFocus">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="descriptionTextEdit">
           <property name="tabChangesFocus">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSmall_items"/>
    <addaction name="actionHighlight_today_entries"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSmall_items">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Small items</string>
   </property>
  </action>
  <action name="actionHighlight_today_entries">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highlight today entries</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EntryListView</class>
   <extends>QWidget</extends>
   <header>entrylistview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateLineEdit</tabstop>
  <tabstop>fromLineEdit</tabstop>
  <tabstop>toLineEdit</tabstop>
  <tabstop>titleTextEdit</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
  <tabstop>comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
